csv/InstalledSoftware.csv

```diff
@@ -1,4 +1,19 @@
 Category,Regex,Comment
+Data Transfer,rclone,
+Data Transfer,megasync,
+Data Transfer,plink,
```

Comment:
plink is part of the PuTTY suite so you probably wont see it in installed applications.

Reply:
No problem, happy for this to be committed and removed